android/festago/app/src/main/java/com/festago/festago/presentation/mapper/ReservationStageMapper.kt

```diff
@@ -2,16 +2,20 @@ package com.festago.festago.presentation.mapper
 
 import com.festago.festago.domain.model.ReservationStage
 import com.festago.festago.presentation.model.ReservationStageUiModel
+import java.time.LocalDateTime
 
 fun List<ReservationStage>.toPresentation() = map { it.toPresentation() }
 
-fun ReservationStage.toPresentation() = ReservationStageUiModel(
-    id = id,
-    lineUp = lineUp,
-    reservationTickets = reservationTickets.map { it.toPresentation() },
-    startTime = startTime,
-    ticketOpenTime = ticketOpenTime,
-)
+fun ReservationStage.toPresentation(): ReservationStageUiModel {
+    return ReservationStageUiModel(
```

Comment:
```suggestion
fun ReservationStage.toPresentation() = ReservationStageUiModel(
```
다른 Mapper들은 다음과 같이 되어서 제안드립니다!